services/shhext/chat/multidevice/service.go

```diff
@@ -7,11 +7,28 @@ import (
 	"github.com/status-im/status-go/services/shhext/chat/protobuf"
 )
 
+type InstallationMetadata struct {
+	// The name of the device
+	Name string `json:"name"`
+	// The type of device
+	DeviceType string `json:"deviceType"`
+	// The FCMToken for mobile devices
+	FCMToken string `json:"fcmToken"`
+}
+
 type Installation struct {
+	// Identity is the string identity of the owner
+	Identity string `json:"identity"`
 	// The installation-id of the device
-	ID string
+	ID string `json:"id"`
 	// The last known protocol version of the device
-	Version uint32
+	Version uint32 `json:"version"`
+	// Enabled is whether the installation is enabled
+	Enabled bool `json:"enabled"`
+	// Timestamp is the last time we saw this device
+	Timestamp int64 `json:"timestamp"`
+	// InstallationMetadata
+	InstallationMetadata *InstallationMetadata `json:"metadata"`
```

Comment:
not clear why it is stored in a separate table, can you add a comment somewhere?

Reply:
I will add a comment, thanks, basically sometimes we have only metadata, and we don't have other info on the device, in such cases, we want to store only the metadata, so I thought it would be best to keep in a separate table.